VTEX - Catalog API.json

```diff
@@ -3334,322 +2998,260 @@
+                                            "description": "Credit that the customer receives when finalizing an order of one specific SKU unit. By filling this field out with `1`, the customer gets U$ 1 credit on the site."
                                         },
-                                        "example": "-8NLU58HJKL2"
-                                    }
-                                }
-                            },
-                            "p3p": {
-                                "content": {
-                                    "text/plain": {
-                                        "schema": {
-                                            "type": "string"
+                                        "EstimatedDateArrival": {
+                                            "type": "string",
+                                            "description": "SKU estimated arrival date in [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format, when the product is on pre-sale. You must take into consideration both the launch date and the freight calculation for the arrival date.",
+                                            "nullable": true
                                         },
-                                        "example": "policyref=\"/w3c/p3p.xml\",CP=\"ADMa OUR NOR CNT NID DSP NOI COR\""
-                                    }
-                                }
-                            },
-                            "powered": {
-                                "content": {
-                                    "text/plain": {
-                                        "schema": {
-                                            "type": "string"
+                                        "ManufacturerCode": {
+                                            "type": "string",
+                                            "description": "Provided by the manufacturers to identify their product. This field should be filled in if the product has a specific manufacturer’s code."
                                         },
-                                        "example": "vtex"
-                                    }
-                                }
-                            },
-                            "x-vtex-operation-id": {
-                                "content": {
-                                    "text/plain": {
-                                        "schema": {
-                                            "type": "string"
+                                        "CommercialConditionId": {
+                                            "type": "integer",
+                                            "description": "Used to define SKU specific promotions or installment rules. In case of no specific condition, use `1` (default value). This field does not accept `0`. Find out more by reading [Registering a commercial condition](https://help.vtex.com/tutorial/registering-a-commercial-condition--tutorials_445)."
```

Comment:
```suggestion
                                            "description": "Commercial condition ID, used to define SKU specific promotions or installment rules. In case of no specific condition, use `1` (default value). This field does not accept `0`. Learn more at [Registering a commercial condition](https://help.vtex.com/tutorial/registering-a-commercial-condition--tutorials_445).",
                                            "default": 1
```